File: api/core/src/commonMain/kotlin/data/UcenjeData.kt
@file:OptIn(ExperimentalJsExport::class)

package data

import domain.Oseba
import domain.Ucenje
import kotlinx.serialization.Serializable
import kotlin.js.ExperimentalJsExport
import kotlin.js.JsExport

@JsExport
@Serializable
data class UcenjeData(
    val ucenje: Ucenje,
    val oseba_refs: Array<Oseba> = arrayOf(),
    val ucenje_ucenec_refs: Array<UcenjeData> = arrayOf(),
)

File: app/src/app/services/api/openapi/models/ucenje-data.ts
/* tslint:disable */
/* eslint-disable */
import { Oseba } from './oseba';
import { Ucenje } from './ucenje';
export interface UcenjeData {
  oseba_refs?: Array<Oseba>;
  ucenje?: Ucenje;
  ucenje_ucenec_refs?: Array<UcenjeData>;
}
